skymap_scanner/server/pixels.py

```diff
@@ -172,7 +172,7 @@ def choose_pixels_to_reconstruct(
     nsides_dict: NSidesDict,
     nside_progression: NSideProgression,
     ang_dist: float = 2.,
```

Comment:
Do we rely on this default? @G-Sommani @tianluyuan @mlincett

Reply:
Maybe we can set here as well `ang_dist` to 3.5?

Reply:
I'd rather the arg not have a default, then we'd rely on the reco_algo instance to supply it. That way we have a single source of truth